### ISSUE TITLE
Custom domain and custom auth don't work together

### ISSUE THREAD

**Issue by edorphy** [Azure/azure-mobile-apps-net-server#236]
My Azure App Service works just fine with custom auth written per the guide documentation.

However when I enable my custom domain I bought through the Azure portal and my SSL Cert to match it, the custom auth falls apart and doesn't work (401).

My default audience and issuer are based upon the hostname (per some tutorials on the subject) are falling back to my .azurewebsites.net host name. When I override this to my custom domain it still doesn't work.

Should this "just work" by design? I haven't found any documentation to say otherwise, that is that custom auth and custom domain don't work together.

This seems similar in description, but it doesn't appear that there was a resolution for Custom Authorization: https://stackoverflow.com/questions/39853261/azure-mobile-app-service-custom-authentication-valid-audiences

**Comment by elamalani** [comment]
@edorphy Sorry for the late follow-up on this issue. Were you able to resolve the problem?

**Comment by edorphy** [comment]
No I didn’t have time to investigate further.  I removed my custom cert and custom domain deployment for the time being. 

I plan to go live with my service this summer so it is certainly something I still want to do. 

I found some forum posts about overriding the website signing key in app settings, so maybe it’s possible to change the issuer and audience that way too?

**Comment by elamalani** [comment]
@edorphy Thanks for the update. Please look at the GitHub readme where we talk about the [future of Azure Mobile Apps](https://github.com/Azure/azure-mobile-apps-js-client#future-of-azure-mobile-apps). At this time, we will only look at fixing critical issues in the product. 

I'll close this issue for now but let us know if you have any questions.

**Comment by edorphy** [comment]
@elamalani The link you added points to the javascript sdk, however I found the same documentation under this project too.

It is my understanding since this has been closed MSFT doesn't believe this is a critical issue. From the consumer perspective it is. I can't be the only client that wants to use a custom domain, secure it properly not using Google/Facebook for login and host it in Azure.

The app service that I've been on a higher tier to support custom domains is a paid upgrade monthly.
The domain that I have acquired through the azure portal for the app service is a paid feature annually.
The custom ssl certificate purchased to match the domain is yet another paid feature annually.

Am I correct in stating that deploying an app service with a custom domain, a custom ssl certificate, and using custom authorization is NOT supported? Not everyone wants to build Facebook Identity into their products these days.
